src/Illuminate/Bus/Dispatcher.php

```diff
@@ -165,9 +167,26 @@ public function dispatchToQueue($command)
      *
      * @param  \Illuminate\Contracts\Queue\Queue  $queue
      * @param  mixed  $command
-     * @return mixed
+     * @return mixed  Optional reference to queued job
+     */
+    public function pushCommandToQueue($queue, $command)
+    {
+        $queueId = $this->pushCommandToQueueWithOptions($queue, $command);
+
+        app(EventDispatcher::class)->dispatch(new JobQueued($queue, $command, $queueId));
+
+        return $queueId;
+    }
+
+    /**
+     * Push the command onto the given queue instance respecting the options set in job.
+     *
+     * @param $queue
+     * @param $command
+     *
+     * @return mixed Optional reference to queued job
      */
-    protected function pushCommandToQueue($queue, $command)
+    protected function pushCommandToQueueWithOptions($queue, $command)
```

Comment:
@jmarcher this is where it is a breaking change. Not really "super-breaking" as it is a protected function but it might create confusion.

Also side note, no I don't copy-paste, i use Github's auto-replies :stuck_out_tongue:

Reply:
So you mean if someone inherits and overwrites this method?

Reply:
There is still a new public method with the name `pushCommandToQueue` doing the same as of before!? Sorry, I don't get it, why it should break.

Reply:
yes in case someone inherits it. No there is no other function. The problem is those using earlier versions will have one function name while those using the newer patch versions will have a different functon name.